release.config.js

```diff
@@ -86,7 +86,7 @@ async function config() {
       ['@semantic-release/github', {
         successComment: getReleaseComment(),
         labels: ['type:ci'],
-        releasedLabels: ['state:released<%= nextRelease.channel ? `-\${nextRelease.channel}` : "" %>']
```

Comment:
Did you manually change this? I think to remember that the `\` was required, because this is not supposed to parsed as a placeholder at this point, but I may be wrong.

Reply:
Maybe we should apply linting only to the `src` folder?

Reply:
Let me verify whether we need the `\`.

Reply:
It seems we can safely remove the `\`, I tested it and the release labels are correctly applied.